packages/edit-post/src/components/preferences-modal/options/enable-custom-fields.js

```diff
@@ -7,10 +7,26 @@ import { Button } from '@wordpress/components';
 import { withSelect } from '@wordpress/data';
 import { store as editorStore } from '@wordpress/editor';
 import { ___unstablePreferencesModalBaseOption as BaseOption } from '@wordpress/interface';
+import { getPathAndQueryString } from '@wordpress/url';
+
+function submitCustomFieldsForm() {
+	const customFieldsForm = document.getElementById(
+		'toggle-custom-fields-form'
+	);
+
+	if ( customFieldsForm ) {
```

Comment:
This check means we'll fail silently if the form is missing from the DOM for whatever reason. Maybe it should fail loudly? i.e. remove the `if`.

Reply:
Ooh, I was too timid! I'll do this in an immediate follow up. Thank you for testing 🙇🏻

Reply:
I could probably move the `customFieldsForm.submit();` out of the condition and test for the referrer only, but I guess we really expect both to be there, right 😄

Reply:
You know what, let's ditch the check now. It wasn't there before. Thanks for the nudge.